tools/dependencies/known-dependencies.txt

```diff
@@ -482,4 +482,5 @@ casdoor-java-sdk-1.11.0.jar
 casdoor-spring-boot-starter-1.6.0.jar
 org.apache.oltu.oauth2.client-1.0.2.jar
 org.apache.oltu.oauth2.common-1.0.2.jar
-snowflake-jdbc-3.13.10.jar
\ No newline at end of file
+snowflake-jdbc-3.13.10.jar
+vertica-jdbc-12.0.4-0.jar
```

Comment:
You should add the license file, see https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/contribute/join/DS-License.md

Reply:
I couldn't find the specific license file, only found `End User License Agreement` at https://www.vertica.com/download/vertica/client-drivers/